### ISSUE TITLE
Use the options design pattern

### ISSUE THREAD

**Issue by JanLenoch** [Kentico/xperience-training-13#13]
### Who

- [ ] Anonymous visitor
- [ ] Patient
- [ ] Doctor
- [ ] Content editor
- [ ] Marketer
- [ ] Administrator
- [x] Developer

### Needs what

To [work with application settings](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/configuration/options?view=aspnetcore-3.1) in a strongly typed manner.

### Why

Not using the options pattern makes the code maintenance error-prone.

### Issue expands upon

This issue expands upon another issue(s): #10 

### Blockers

This issue is blocked by the following issue(s): #`issue number`
